src/vaadin-contextmenu-items-mixin.html

```diff
@@ -236,7 +243,7 @@
     __initMenu(root, menu) {
       if (!root.firstElementChild) {
         root.innerHTML = `
-          <vaadin-context-menu-list-box></vaadin-context-menu-list-box>
+          <vaadin-context-menu-list-box theme="${this.theme}"></vaadin-context-menu-list-box>
```

Comment:
Make sure the theme attribute isn't "null" or "undefined" or event "" at any time.

Reply:
Updated.